source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+ - Fix null pointer dereference of param in [3a44c78f14](https://github.com/php/php-src/commit/3a44c78f14) by Niels Dossche
+ - Improve mb_detect_encoding's recognition of Turkish text in [f40c3fca88](https://github.com/php/php-src/commit/f40c3fca88) by Alex Dowad
+ - Fix memory leak in posix_ttyname() in [GH-10190](https://github.com/php/php-src/pull/10190) by George Peter Banyard 💜
+ - Fix [GH-10187](https://github.com/php/php-src/issues/10187): Segfault in stripslashes() with arm64 in [GH-10188](https://github.com/php/php-src/pull/10188) by Niels Dossche
+ - Fix variation5-win32(-mb).phpt wrt. parallel test execution in [GH-10189](https://github.com/php/php-src/pull/10189) by Christoph M. Becker
+ - Better document constructors in [eebf3bc0ba](https://github.com/php/php-src/commit/eebf3bc0ba) by Máté Kocsis 💜
+ - Do not display non-existent constructors in [38e138798d](https://github.com/php/php-src/commit/38e138798d) by Máté Kocsis 💜
+ - Do not list private constructors as inherited in [7b08fe9f2d](https://github.com/php/php-src/commit/7b08fe9f2d) by Máté Kocsis 💜
+ - Fix [GH-10200](https://github.com/php/php-src/issues/10200): zif_get_object_vars: Assertion `!(((__ht)->u.flags & (1<<2)) != 0)' failed in [GH-10209](https://github.com/php/php-src/pull/10209) by Niels Dossche
+ - Fix [GH-10200](https://github.com/php/php-src/issues/10200): zif_get_object_vars: Assertion `!(((__ht)->u.flags & (1<<2)) != 0)' failed in [GH-10209](https://github.com/php/php-src/pull/10209) by Niels Dossche
+ - ext/opcache/zend_shared_alloc: add assertions on "locked" flag in [e1a25ff2ed](https://github.com/php/php-src/commit/e1a25ff2ed) by Max Kellermann
+ - ext/opcache/zend_shared_alloc: change "locked" check to assertion in [10d43c40dd](https://github.com/php/php-src/commit/10d43c40dd) by Max Kellermann
+ - Fix [GH-10202](https://github.com/php/php-src/issues/10202): posix_getgr(gid|nam)_basic.phpt fail in [d5f0362e59](https://github.com/php/php-src/commit/d5f0362e59) by Niels Dossche
+ - Implement mb_output_handler using fast text conversion filters in [a9a672048b](https://github.com/php/php-src/commit/a9a672048b) by Alex Dowad
+ - Implement mb_str_split using fast text conversion filters in [88c99afdac](https://github.com/php/php-src/commit/88c99afdac) by Alex Dowad
+ - Mark default interned strings as valid UTF-8 where appropriate in [e2654a532a](https://github.com/php/php-src/commit/e2654a532a) by Alex Dowad
+ - Implement php_mb_zend_encoding_converter using fast text conversion filters in [953864661a](https://github.com/php/php-src/commit/953864661a) by Alex Dowad
+ - Implement mb_detect_encoding using fast text conversion filters in [0e7160b836](https://github.com/php/php-src/commit/0e7160b836) by Alex Dowad
+ - Use smart_str in mb_http_input rather than mbfl_memory_device in [3b5072f6f6](https://github.com/php/php-src/commit/3b5072f6f6) by Alex Dowad
```

Comment:
```suggestion
 - Use `smart_str` in `mb_http_input` rather than `mbfl_memory_device` in [3b5072f6f6](https://github.com/php/php-src/commit/3b5072f6f6) by Alex Dowad
```